src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageEndpoint.cs

```diff
@@ -18,13 +19,12 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
-    internal class RazorLanguageEndpoint : IRazorLanguageQueryHandler, IRazorMapToDocumentRangesHandler, IRazorMapToDocumentEditsHandler
+    internal class RazorLanguageEndpoint :
```

Comment:
I feel like it was a mistake on my part to make RazorLanguageEndpoint so "all-encompassing". I'd actually break the diagnostics handling into its own endpoint class